examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -173,6 +173,17 @@ function implicit_tendency_special!(Yₜ, Y, p, t)
     @nvtx "implicit tendency special" color = colorant"yellow" begin
         Fields.bycolumn(axes(Y.c)) do colidx
 
+            # this assumes that thermodynamic covariances are diagnostic
+            if p.tendency_knobs.has_turbconv
+                @. Yₜ.c.turbconv.up[1].ρarea[colidx] = FT(0)
+                @. Yₜ.c.turbconv.up[1].ρaθ_liq_ice[colidx] = FT(0)
```

Comment:
Does the `[1]` assume we only have one updraft?

Reply:
Yes

Reply:
Does
```julia
@. Yₜ.c.turbconv[colidx] = FT(0)
@. Yₜ.f.turbconv[colidx] = FT(0)
```
work? I imagine that would be ideal

Reply:
yes that would work